rs/rosetta-api/icp/src/ledger_client.rs

```diff
@@ -137,6 +138,18 @@ pub enum OperationOutput {
     ListNeuronsResponse(ListNeuronsResponse),
 }
 
+impl TryFrom<ObjectMap> for OperationOutput {
+    type Error = ApiError;
+    fn try_from(o: ObjectMap) -> Result<Self, Self::Error> {
```

Comment:
No issue here, just for my own learning: why do we use `Self` here but in neuron_response.rs:34 we use `NeuronResponse` specifically?

Reply:
It should not make any difference.